lib/Hooks/ImageHooks.php

```diff
@@ -49,23 +51,10 @@ public function register() {
 	public function postCreate(Node $node) {
 		$absolutePath = $this->dataDirectory.$node->getPath();
 
-		$dimensions = getimagesize($absolutePath);
+		$metadataExtracter = new ExtractMetadata($absolutePath);
```

Comment:
Sure, I've mentioned how it works in the ImageHooks PR I think, but here is the official documentation which might be quicker to understand:
https://docs.nextcloud.org/server/9/developer_manual/app/container.html